misc/python/materialize/feature_benchmark/executor.py

```diff
@@ -73,6 +78,53 @@ def Kgen(self, topic: str, args: List[str]) -> Any:
     def DockerMem(self) -> int:
         return self._composition.mem("materialized")
 
+    def Messages(self) -> Optional[int]:
+        """Return the sum of all messages in the system from mz_internal.mz_message_counts_per_worker"""
+
+        def one_count(e: Docker) -> Optional[int]:
+            result = e._composition.sql_query(
+                dedent(
+                    """
```

Comment:
Yes, the repro I created blows things up significantly by using updates to the skewed keys. And indeed, it focuses only on the join region, so the counts do not get diluted by other operators.

But I can see that in your case it is more consistent here to use total counts, since the feature benchmark uses aggregate measurements like the total elapsed time for the scenario and the total memory for the container.

By focusing on messages `sent` and the `from_operator_id`, we are looking at outputs of operators only, so this will be more effective at detecting if some operator starts emitting much more than it should.